help_function/help_function.xml

```diff
@@ -121,6 +121,7 @@
     <test testName="onedpl_test_stable_partition" configFile="config/TEMPLATE_help_function_skip_cuda_backend.xml" />
     <test testName="onedpl_test_tabulate" configFile="config/TEMPLATE_help_function_skip_cuda_backend.xml" />
     <test testName="onedpl_test_transform_if" configFile="config/TEMPLATE_help_function_skip_cuda_backend.xml" />
+    <test testName="onedpl_test_transform_output_iterator" configFile="config/TEMPLATE_help_function_skip_cuda_backend.xml" />
```

Comment:
This test internally uses `sycl::malloc_shared`. Is it safe to use the `config/TEMPLATE_help_function_skip_cuda_backend.xml/` or should we use `config/TEMPLATE_help_function_usmnone.xml`?

Reply:
I've rewritten the tests to not rely upon USM memory.